.teamcity/_self/projects/WebApp.kt

```diff
@@ -264,21 +264,13 @@ object RunAllUnitTests : BuildType({
 			"""
 		}
 		bashNodeScript {
-			name = "Build components storybook"
+			name = "Run storybook tests"
```

Comment:
I merged the two Storybook jobs into one, and added `composite-checkout`.

Also, I dropped `NODE_ENV="production"`. It was causing problems, because Storybook sets it to `development` internally, and webpack's `DefinePlugin` was failing with https://github.com/storybookjs/storybook/issues/14257#issuecomment-877876022

Reply:
Makes sense